templates/skin.mustache

```diff
@@ -63,8 +63,8 @@
 		<section class="skin-wmau-footer-left">
 			<p class="skin-wmau-image-logo-black-small"></p>
 			<p class="skin-wmau-contact">
-				<strong>Wikimedia Australia, Inc.</strong><br>
```

Comment:
This is the formal registered name of WMAU, which is different from the site title (we don't want `, Inc.` appearing everywhere). I guess a new message is required for this? Or a config var?

Reply:
Message is probably the way to go just because you won't need to do any additional PHP wire up. You can set that in i18n/en.json or interface messages... up to you!